sql/moz-fx-data-shared-prod/braze_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Braze Derived
+description: |-
+  Dataset with tables that combine source data
+  as a source of truth for transformations into the braze_external dataset.
+dataset_base_acl: restricted
```

Comment:
```suggestion
dataset_base_acl: derived_restricted
```

Reply:
Does this automatically generate the dataset with `derived_restricted`?
In the [documentation](https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#creating-a-new-bigquery-dataset) it only states that `derived` and `view` are possible options ... and the dataset is not here yet

Reply:
dataset is there now \o/